backend/clap/src/main/java/zpi/ppea/clap/security/FirebaseAgent.java

```diff
@@ -12,64 +12,56 @@
 import lombok.Data;
 import lombok.RequiredArgsConstructor;
 import lombok.SneakyThrows;
-import org.apache.logging.log4j.LogManager;
-import org.apache.logging.log4j.Logger;
+import lombok.extern.slf4j.Slf4j;
 import org.springframework.stereotype.Service;
-import zpi.ppea.clap.exceptions.NoAccessToResource;
+import zpi.ppea.clap.config.ValueConfig;
+import zpi.ppea.clap.exceptions.GrpcDatastoreException;
 import zpi.ppea.clap.exceptions.UserNotAuthorizedException;
 import zpi.ppea.clap.service.DataStoreService;
 
 import java.util.HashMap;
 import java.util.Map;
 import java.util.Optional;
 
+@Slf4j
 @Service
 @RequiredArgsConstructor
 public class FirebaseAgent {
 
-    private final Logger log = LogManager.getLogger("firebase-app");
     private final DataStoreService client;
+    private final ValueConfig valueConfig;
+    public static final String REFRESH_TOKEN_NAME = "refresh";
```

Comment:
that's not what this `"refresh"` value is. It's a header name, not token name - or `REFRESH_TOKEN_HEADER_NAME` if you insist

Reply:
my mistake, I will correct it